src/main/java/ru/rosatom/documentflow/dto/DocumentDto.java

```diff
@@ -61,5 +61,8 @@ public class DocumentDto {
 
     @Schema(description = "Статус")
     String finalDocStatus;
+
+    @Schema(description = "Комментарии")
+    private List<DocProcessCommentDto> comment;
```

Comment:
не забывай множественное число
```suggestion
    @Schema(description = "Комментарии")
    private List<DocProcessCommentDto> comments;
```